landoapi/hg.py

```diff
@@ -286,6 +313,28 @@ def apply_patch(self, patch_io_buf):
                 + ["--logfile", f_msg.name]
             )
 
+    def format(self) -> Optional[List[str]]:
+        """Run `hg fix` to format the currently checked-out stack."""
+        try:
+            logger.info("running `hg fix`")
```

Comment:
`self.run_hg` already logs the command being run (same applies to `update` below)